content/Hardware/Generic/How-to-do-a-loopback-test.md

```diff
@@ -24,6 +22,12 @@ A failed loopback test indicates a problem with one or more of the following:
 
 4. Connect a jumper cable from **Digital 0 (RX)** to **Digital 1 (TX)**.
 
+   ![](img/loopback-circuit-uno.png)
+   _Pin connections (orange) for Arduino UNO._
```

Comment:
The pin connection looks confusing as it looks like all 4 pins (reset, 3v3, 5v, and gnd) have to be shorted together.

Reply:
Fixed